### PR TITLE
Small optimisations

### PR DESCRIPTION
**Pull request by NDevox**
Making small changes to source code as I'm reading through it which I feel may be appropriate/useful or enhance the readability of the code.

This is a prospective pull - happy for it to be rejected as most changes are very minor.

### CODE REVIEW

**Comment by NDevox** [comment]
Trying to figure out why this build is failing but not getting anywhere. Any ideas?

**Comment by NDevox** [comment]
Excess lines apparently.

File: elastalert/util.py [hunk 1]
```diff
@@ -47,8 +47,7 @@ def lookup_es_key(lookup_dict, term):
         subkey = ''
 
         while subkeys:
-            subkey += subkeys[0]
-            subkeys = subkeys[1:]
+            subkey = subkeys.pop(0)
```
Comment (Qmando) [inline elastalert/util.py]: Should be ``subkey += subkeys.pop(0)``.
For example, if subkeys is ['foo', 'bar'], we want to test 'foo' then 'foo.bar'.

Comment (NDevox) [inline elastalert/util.py]: @Qmando Good catch, fixed now.

**Comment by fiunchinho** [comment]
Looks good to me

**Pull request merged**